include/networkit/auxiliary/BucketPQ.hpp

```diff
@@ -56,6 +58,8 @@ class BucketPQ: public PrioQueue<int64_t, index> {
      */
     void construct(uint64_t capacity);
 
+    std::pair<int64_t, index> extractMin(bool removeElement);
```

Comment:
Is this signature (in particular, returning a pair) in line with the other priority queues?

Reply:
Yes, `PrioQueue::extractMin` returns a key-value pair.